CHANGELOG

```diff
@@ -1,5 +1,110 @@
 This file contains a summary of changes to the Oppia code base.
 
+v3.0.5 (02 Dec 2020)
+------------------------
+Contributor Pages
+* Fix part of #10687: Make it possible to get the top 5 suggestions that have been waiting longest for review per reviewer. (#10860)* Fix #10955: Make the updates to the CommunityContributionStatsModel transactional in the PopulateContributionStatsOneOffJob (#10956)
```

Comment:
```suggestion
* Fix part of #10687: Make it possible to get the top 5 suggestions that have been waiting longest for review per reviewer. (#10860)
* Fix #10955: Make the updates to the CommunityContributionStatsModel transactional in the PopulateContributionStatsOneOffJob (#10956)
```

There appears to be a formatting issue here.

Reply:
Done.